EXPLAINER.md

```diff
@@ -53,15 +53,17 @@ with multiple screens. Here are some use cases that inform the goals below:
 ## Goals
 
 The following specific goals and proposed solutions make incremental extensions
-of existing window placement APIs to support expanded multi-screen environments. 
+of existing window placement APIs to support extended multi-screen environments.
 * Support requests to show elements fullscreen on a specific screen
   * Extend `Element.requestFullscreen()` for specific screen requests
 * Support requests to place web app windows on a specific screen
   * Extend `Window.open()` and `moveTo()/moveBy()` for cross-screen coordinates
 * Provide requisite information to achieve the goals above
-  * Add `isMultiScreen()` to expose whether a device has multiple screens
-  * Add `getScreens()` to expose information about available screens
-  * Add a `screenschange` event, fired on screen connection or property changes
+  * Add `Screen.isExtended` to expose the presence of extended screen areas
+  * Add `Screen.onchange`, an event fired when Screen attributes change
```

Comment:
I assume the event is called "change" but there is an `attribute EventHandler onchange` that can be used in addition to addEventListerner("change", ....)